README.md

```diff
@@ -198,7 +198,7 @@ Example:
 
 <!-- AUTHOR -->
 
-## 👥 Authors <a name="author"></a>
+## 👥 Author <a name="author"></a>
```

Comment:
- [ ] Good job on using the updated README template 👏 👏 However you have not added all the important information. There is just placeholder text. Please go back and edit it, and add customized content in each section 😃 👍 Remember to add also the project name 😉

![image](https://user-images.githubusercontent.com/90222110/203616520-1e5042d4-f117-43b5-ab34-cb451ba360a6.png)

![image](https://user-images.githubusercontent.com/90222110/203616637-80271ac3-83d6-4b31-9295-e99f73a29813.png)